students/hirot_directory/session03/mailroom.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+donor_names = ["William Gates III", "Mark Zuckerberg", "Jeff Bezos", "Paul Allen"]
+donations = [[200,500], [2000,3000], [4000,5000], [100,150]]
+
+
+def thank_you():
+
+	while True:
+
+		if (str(input("Do you want to see the donor list? if yes, type List: ")) == "list"):
+			print(donor_names, donations)
+		new_donor = str(input("Full Name: "))
+		new_donation = int(input("How much do you want to donate?: "))
+	
+		if (new_donor not in donor_names):
+			donor_names.append(new_donor)
+			donations.append([new_donation])
+			print("Thank you for donation, {}! Your donation amount is {}.".format(new_donor, new_donation))
+
+		else:
+			number = donor_names.index(new_donor) #4
+			donations[number].append(new_donation)
+			print("Thank you donation, {}! Your donation amount is {}.".format(new_donor, new_donation))
+			
+		for index, name in enumerate(donor_names):
+			donation = donations[index]
+			statement = '{} contributed ' + '{}, ' * len(donation)
+			print(statement.format(name, *donation))
+
+		if(input("Type 'exit' to end?").lower() == "exit"): break
+
+
+def print_report():
+
+
+	print("{msg1: <50}| {msg2: <12}| " \
+
+      "{msg3:<10}| {msg4: <12}".format(msg1="Donor Name",
```

Comment:
using keyword arguments for formatting is great, but it's mostly useful if you use meaningful names, like "name", "avg_donation", etc....

This way, you might as well, just use positional arguments.

Reply:
I see. This portion is what I borrowed the code from other students that posted on Slack (Eowyn?) I will note so.